packages/pcd-types/src/pcd.ts

```diff
@@ -1,74 +1,165 @@
+/**
+ * All PCDs consist of a "claim", which is the human-interpretable statement
+ * that the PCD is making (i.e. "I am a Zuzalu resident"); and a "proof" attached
+ * to the "claim," which is a cryptographic or mathematical proof of the claim.
+ * All PCDs within this SDK also expose a `prove` and `verify` function, which
+ * allow you to instantiate them, and verify that they are indeed correct.
+ */
 export interface PCD<C = unknown, P = unknown> {
+  /**
+   * Uniquely identifies this instance. PCDpass cannot have more than one
+   * {@link PCD} with the same id.
+   */
   id: string;
-  type: string;
-  claim: C;
-  proof: P;
-}
 
-export interface SerializedPCD<_T extends PCD = PCD> {
+  /**
+   * Refers to {@link PCDPackage#name} - each {@link PCD} must come from a
+   * particular {@link PCDPackage}.
+   */
   type: string;
-  pcd: string;
-}
 
-/**
- * This interface can be optionally returned by the package ƒor any given
- * PCD, which allows the package some degree of control over how the PCD
- * is displayed in the passport application.
- */
-export interface DisplayOptions {
   /**
-   * Shown to the user in the main page of the passport, where they can
-   * see all of their cards. If `header` is undefined, the passport will use
-   * `renderCardBody` with `returnHeader` set to true.
+   * Human- or end-user-application-interpretable data.
    */
-  header?: string;
+  claim: C;
 
   /**
-   * Shown to the user in the `GenericProveScreen`, allowing them to
-   * disambiguate between different pcds of the same type. In the future,
-   * we'll have a better way to disambiguate between them.
+   * A cryptographic proof of the {@link PCD#claim}.
    */
-  displayName?: string;
+  proof: P;
 }
 
+/**
+ * Each type of {@link PCD} has a corresponding {@link PCDPackage}. The
+ * {@link PCDPackage} of a {@link PCD} type defines the code necessary to
+ * derive meaning from and operate on the data within a {@link PCD}.
+ *
+ * @typeParam {@link C} the type of {@link PCD.claim} for the {@link PCD} encapsulated
+ *   by this {@link PCDPackage}
+ *
+ * @typeParam {@link P} the type of {@link PCD.proof} for the {@link PCD} encapsulated
+ *   by this {@link PCDPackage}
+ *
+ * @typeParam {@link A} - the type of the arguments passed into {@link PCDPackage#prove}
```

Comment:
There are definitely two concepts here, and I'm not happy about it. I've been tracking this in the back of my mind for a few months now. It's the sort of thing that is annoying but I can never quite get to because my hands are full. Maybe we should keep a running document of potential improvements to the PCD type system and knock them out all in one go for a version `v0.2` (where the current version is `v0.1)